app/controllers/static_controller.rb

```diff
@@ -0,0 +1,6 @@
+class StaticController < ApplicationController
+  def index
+    @greetings = Greeting.all.pluck(:greeting)
+    render json: { greetings: @greetings.sample }
+  end
```

Comment:
- [ ] When you run the dev server, The app shows JSON data rendered from the API endpoint you created. That is not supposed to happen because the home page is supposed to render a random greeting using a React app as a front-end and an API endpoint created by Rails. Please fix that.

[Hint] To fix this issue:
1. Create a `greetings_controller.rb` and include all this code inside then⬇️
2. Remove any code from the index method to be like this⬇️
```suggestion
  def index; end
```
3. Edit the static index view to render the react app you created.